consensus/types/src/beacon_block.rs

```diff
@@ -1122,5 +1255,28 @@ mod tests {
             BeaconBlock::from_ssz_bytes(&bad_block.as_ssz_bytes(), &spec)
                 .expect_err("bad electra block cannot be decoded");
         }
+
+        // BeaconBlockFulu
+        {
+            let good_block = BeaconBlock::Fulu(BeaconBlockFulu {
+                slot: fulu_slot,
+                ..<_>::random_for_test(rng)
+            });
+            // It's invalid to have a Fulu block with a epoch lower than the fork epoch.
+            let _bad_block = {
+                let mut bad = good_block.clone();
+                *bad.slot_mut() = electra_slot;
+                bad
+            };
+
+            assert_eq!(
+                BeaconBlock::from_ssz_bytes(&good_block.as_ssz_bytes(), &spec)
+                    .expect("good fulu block can be decoded"),
+                good_block
+            );
+            // TODO(fulu)
+            //BeaconBlock::from_ssz_bytes(&bad_block.as_ssz_bytes(), &spec)
+            //    .expect_err("bad fulu block cannot be decoded");
```

Comment:
Why is this commented out? is this failing

Reply:
Ah yeah, I forgot to add a comment explaining this. Since the method we use to invalidate the blocks is by setting the slot to an epoch before the fork epoch, it actually becomes a valid Electra block (since Fulu blocks don't yet contain any new fields) and thus doesn't error on the decode.

I figured the easiest way was just wait to uncomment this out until Fulu has actual features. Otherwise we could rewrite the way we test invalid block decoding.

Reply:
Oh, I incorrectly assumed the slot would invalidate the decoding, got it now - thanks!
Yeah i think it's fine to leave it as it is with a comment 👍

Reply:
I've added a comment now, let me know if you think this is unreasonable :+1:

Reply:
Nice, thanks